docs/index.rst

```diff
@@ -1,38 +1,22 @@
-scipp - Multi-dimensional data arrays with labeled dimensions
-=============================================================
+Scipp -- Multi-dimensional data arrays with labeled dimensions
+==============================================================
 
 .. raw:: html
 
    <span style="font-size:1.2em;font-style:italic;color:#5a5a5a">
       A Python library enabling a modern and intuitive way of working with scientific data in Jupyter notebooks
    </span>
 
-|data-structures| |plotting| |masking|
-|binning| |slicing| |scipp-neutron|
+|repr-html| |filtering| |show-svg|
 
-.. |data-structures| image:: _static/title-repr-html.png
+.. |repr-html| image:: _static/title-repr-html.png
```

Comment:
Can we make all of the figures SVG?

Reply:
Why?

Reply:
Would potentially look better on high-res monitors? The cube representation is already an svg and we convert it to png instead of using the svg directly.

Reply:
Looks fine on my screen. But sure, can you do that please, the figure is from here: https://scipp.github.io/tutorials/from-tabular-data-to-binned-data.html#From-event-based-metadata-to-bin-based-metadata.